src/EventStore.Projections.Core.Tests/Services/projections_manager/when_deleting_a_persistent_projection_with_default_options.cs

```diff
@@ -39,6 +42,9 @@ yield return
 
 		[Test, Category("v8")]
 		public void a_projection_deleted_event_is_written() {
+			var deletedStreamEvents =_consumer.HandledMessages.OfType<ClientMessage.DeleteStream>().ToList();
+
+
```

Comment:
Unnecessary double empty lines.

Reply:
@YoEight - sorry I only noticed your comments regarding the squash.
Up until recently, I hadn't used squash before, so for this PR I used Visual Studio.
I selected the commits, right clicked and hit squash.
I pushed this up (the CI process kicked off, so assumed it was ok)

Reply:
![image](https://user-images.githubusercontent.com/13871941/125599263-41d45482-61f3-442e-96de-1dbe8fa82616.png)

Reply:
![image](https://user-images.githubusercontent.com/13871941/125599433-76fb3fc3-ec21-4bdf-8314-87c4779a6333.png)

Reply:
From what I can tell, it took the two previous Commits, and turned them into a single Commit.